libs/hdf-converters/src/sonarqube-mapper.ts

```diff
@@ -115,14 +125,18 @@ export class SonarQubeResults {
     while (paging) {
       await axios
         .get<IssueData>(`${this.sonarQubeHost}/api/issues/search`, {
-          auth: {username: this.userToken, password: ''},
+          auth: { username: this.userToken, password: '' },
           params: {
             componentKeys: this.projectId,
             types: 'VULNERABILITY',
-            p: page
+            p: page,
+            ...(this.branchName ? { branch: this.branchName } : {}),
```

Comment:
instead of doing the ternary operator you can use the AND operator (&&)

a small article on it: https://medium.com/@slamflipstrom/conditional-object-properties-using-spread-in-javascript-714e0a12f496

i'd prefer to use this form instead of the ternary operator so as to match the other mappers